streampipes-extensions/streampipes-processors-geo-jvm/src/main/resources/org.apache.streampipes.processors.geo.jvm.jts.processor.buffergeometry/documentation.md

```diff
@@ -0,0 +1,96 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+  -->
+
+## Buffer Geometry
+
+<p align="center">
+    <img src="icon.png" width="150px;" class="pe-image-documentation"/>
+</p>
+
+***
+
+## Description
+
+Creates a buffer polygon geometry from a geometry
+***
+
+## Required inputs
+
+* WKT of a JTS Geometry
+* Integer value representing EPSG code
+* Distance
+* Cap Style
+* Join Style
+* Mitre-Limit
+* Side
+* Simplify Factor
+* Quadrant Segments
+***
+
+## Configuration
+
+### 1st parameter
+Input Geometry
+
+### 2nd parameter
+EPSG code value
+
+### 3rd parameter
+Distance
+The buffer distance around in geometry in meter
+
+### 4rd parameter
+Cap Style
```

Comment:
What do you think of using the name as the actual caption instead of `3rd parameter` etc.?

Reply:
You are correct. I just pasted and copied it from an older document structure, and I should have used this one  https://cwiki.apache.org/confluence/display/STREAMPIPES/04+Additionals#id-04Additionals-documentationtemplate instead

Reply:
I just recognized that this structure is also used in other (geo)  processors. So it's also a change request to other processors that I'm keeping an eye on.

Reply:
improved